### ISSUE TITLE
ENA9

### ISSUE THREAD

**Issue by ManpreetBahl** [ShabadOS/database#1380]
| Key | Value |
| --- | ----- |
View | https://database.shabados.com/sources/1/page/701/line/32
Source | Sri Guru Granth Sahib Ji
Page | 701
ID | ENA9
Line | mnu qnu inrml krq ikAwro; hir isMcY suDwsM joir ]

`suDwsM joir` needs to be corrected to `suDw sMjoir`
![image](https://user-images.githubusercontent.com/48495143/73425829-a9b03f00-42e7-11ea-9fc6-8a7c839fe0da.png)

Source Link for Screenshot above: https://web.archive.org/web/20171118031846/http://old.sgpc.net/CDN/Siri%20Guru%20Granth%20Sahib%20without%20Index%20%28Uni%29.pdf

**Comment by Sarabveer** [comment]
@ManpreetBahl Thanks for the correction submission!

Depends #95

**Comment by bhajneet** [comment]
I've increased the need for alternates allowing for full SGPC parity. This should be one of the first things completed in February @Harjot1Singh 

@ManpreetBahl as Sarabveer said, thank you very much for helping out :) I see you used the new issue template which is a bit outdated -- going to remove that so others aren't confused.

In the future, do basically what you did, but go through the DB Viewer, it should be easier. Please see our [tutorial](https://tutorials.shabados.com/tutorials/1.0.0/database-viewer/proofreading.html#_open_an_issue_reference_ticket), if it helps (you may always add more information than the tutorial suggests, more info = better!)

And of course if you'd like to help out in any other way please get in touch, we can always use a good volunteer if you have the time and motivation 👍

**Comment by ManpreetBahl** [comment]
@bhajneet Will keep in mind for next time.
I'm also willing to help out as much as I can. 😎👍

**Comment by bhajneet** [comment]
Depends on #1480

**Comment by bhajneet** [comment]
![image](https://user-images.githubusercontent.com/14130567/77808577-5138bc00-7062-11ea-9833-b05aea941ac5.png)

**Comment by bhajneet** [comment]
@ManpreetBahl I think we have to close the original PR here and redo off dev branch

**Comment by ManpreetBahl** [comment]
Sure thing!

**Comment by bhajneet** [comment]
https://github.com/ShabadOS/database/blob/dev/data/Sri%20Guru%20Granth%20Sahib%20Ji/0701.json#L1589

This is the new line to adjust.
